packages/blockchain/flow/src/lib/services/flow.tx.ts

```diff
@@ -488,40 +484,40 @@ const _sendTransaction = async (
       fcl.limit(1000),
     ])
   } catch (e) {
+    console.error('Sending transaction with Flow client failed:', e)
```

Comment:
Please take a look at approach
`btc.sdk.errors.ts` applied in  `btc.tx.ts` used in tests `btc.tx.spec.ts`